README.md

```diff
@@ -18,6 +18,91 @@ Using the non-normalized data in the **BlogPost** table below, create a database
 
 Fork this repository and add your tables here:
 
+### BlogPost
+| BlogPostId | Slug                          | Title                                | Body           |
+|------------|-------------------------------|--------------------------------------|----------------|
+| 1          | serilog-structured-logging    | Structured Logging with Serilog      | Long Text Here |
+| 2          | persisting-smart-enum         | Persisting a Smart Enum with EF Core | Long Text Here |
+| 3          | localization-aspnet-core-apis | Localization in ASP.NET Core APIs    | Long Text Here |
+
+
+### Author
+| AuthorId | AuthorName    | AuthorEmail                 | AuthorBio      |
+|----------|---------------|-----------------------------|----------------|
+| 1        | Eric Fleming  | ericfleming@nimblepros.com  | Long Text Here |
+| 2        | Kyle McMaster | kylemcmaster@nimblepros.com | Long Text Here |
+| 3        | Steve Smith   | steve@nimblepros.com        | Long Text Here |
+
+
+### BlogPostAuthor
+| BlogPostId | AuthorId |
+|------------|----------|
+| 1          | 1        |
+| 1          | 2        |
+| 2          | 3        |
+| 3          | 2        |
```

Comment:
Nice support for multiple authors. What are your thoughts on how you would deal with blogs without authors and/or authors without blogs in the application (not here in the database)?

Reply:
Not sure I understand the question exactly but I'd add a validation check when creating/updating the blog to ensure that there is at least one author.  If authors can't exist unless they have a blog, then when deleting a blog I'd check that the author is still linked to at least one other blog otherwise the author would be deleted as well.

Reply:
Right that's all I was wondering. It would depend on the app and its requirements, for sure.